contracts/src/periphery/cert/EigenDACertTypes.sol

```diff
@@ -0,0 +1,21 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.9;
+
+import {EigenDATypesV1 as DATypesV1} from "src/core/libraries/v1/EigenDATypesV1.sol";
+import {EigenDATypesV2 as DATypesV2} from "src/core/libraries/v2/EigenDATypesV2.sol";
+
+/// @title EigenDACertTypes
+/// @notice This library defines the types for each EigenDA certificate version.
+/// @dev It is required that RBN be located in positions 64:96 (padded) in the ABI encoded certificate.
+///      The reason for this is for backwards compatibility with EigenDACertVerifierV2.
+library EigenDACertTypes {
+    /// @dev This struct is the same as the types used in CertVerifierV2's verifyDACertV2 function.
```

Comment:
Not technically true since the V2 cert uses a different field ordering

Reply:
I don't think so? https://github.com/Layr-Labs/eigenda/blob/fb43113fe02d8b42882d14aaa7b94b8183a1cffd/contracts/src/periphery/cert/v2/EigenDACertVerifierV2.sol#L72 i think it is the same.

Reply:
referring to the offchain cert :(

Reply:
oh right. can that not be handled off chain somehow? I think it is confusing for the contracts to mention this because it was never used that way by any contracts at all.

Reply:
I agree with removing the comment because it is not really relevant to anyone but us.